.cursor/mcp.json

```diff
@@ -6,7 +6,7 @@
         "run",
         "--rm",
         "-i",
-        "safedep/pinner-mcp:latest"
+        "ghcr.io/safedep/pinner-mcp:latest"
```

Comment:
The local MCP.json file is for local development. When we are developing this project we need a way to build the container and use it locally.